src/aznfswatchdog

```diff
@@ -221,14 +225,18 @@ reconcile_conntrack()
 
     if [ -n "$output" ]; then
         for entry in $output; do
-            # XXX Remove this log after running for few days.
+            # XXX Remove this log after running for few days.(not too noisy!)
             vecho "$entry"
 
-            matchstr=".* SYN_SENT src=.* dst=$l_ip sport=([0-9]+) dport=([0-9]+).*"
+            # pasting here for ease during review (remove later)
```

Comment:
instead of this, say

`#` Sample conntrack entry.

and we can leave it around